design/FY2017/MultipleAirloopsInOneZone.md

```diff
@@ -0,0 +1,546 @@
+
+### 2. Allow more than one return air node in a zone ###
+Change the `Zone Return Air Node Name` field to allow a NodeList, similar to the way that zone inlet nodes are input.
+
+### 3. Add a new field to AirloopHVAC to specify loop return air flow fraction
+To model pressurized systems and to help allocate return flows among multiple airloops, it is proposed that a new field be added to control the loop return flow rate as a fraction of the loop supply flow rate.
+
+### 4. Revise return air flow and air loop flow balance calculations
+Currently, the zone return air node is passive and receives whatever flow is left over from total supply less total unbalanced exhaust for that zone.  There is also an option to control the return air flow using the `Zone Return Air Flow Rate Fraction Schedule Name` and `Zone Return Air Flow Rate Basis Node or NodeList Name` fields. Also, each airloop checks the total supply vs unbalanced exhaust on the loop and balances the loop by adjusting return node flows proportionally to match.  The heart of these calculations is ZoneEquipmentManager::CalcZoneMassBalance.
+
+The return air flow will be allocated as follows, similar to the current return air flow calculations with some added or modified steps:
+
+  - Step 1 - Set known return node flows
+	  - Set any return nodes that have a flow rate specified in the ZoneHVAC:EquipmentConnections object. (*existing step*)
+	  - For an air loop without an outdoor air system, set the return node flow equal to the corresponding supply air inlet multiplied by the system return flow fraction (input value from AirloopHVAC). (*new step*)
+  - Step 2 - Calculate remaining unallocated return air flow for each zone (*modified step*)
+      - UnallocatedReturn = TotalSupply - UnbalancedExhaust - KnownReturnFlows
+  - Step 3 - Set remaining return node flows (*new step*)
+      - Set remaining return node flows proportional to the supply flow to that zone from the corresponding airloop.  
+      - For example, if Zone 1 is supplied with 0.8 m3/s from Air Loop A and 0.2 m3/s from Air Loop B, then allocate 80% of the remaining return flow to the return air node connected to Loop A and 20% to Loop B.
+  - Step 4 - Allocate unbalanced exhaust air flows to each air loop (*modified step*)
+      - For a given zone, the unbalanced exhaust air flow is distributed to any air loop that has an outdoor air system in proportion to the loop's supply flow to that zone. Air systems without an outdoor air inlet are excluded.
+  - Step 5 - Balance each air loop (*existing step*)
+      - LoopReturn0 = Sum of Return Node flow rates on the loop after Steps 1-4.
+      - LoopReturn = LoopSupply - LoopExhaust
+      - For each return node on the loop AdjustedReturnFlow = ReturnFlow*LoopReturn/LoopReturn0
+
+### 5. Allocate return air heat gains to specific return air nodes
+This applies to return air heat gain from lights, refrigerated case under-case return,and airflow windows.
+
```

Comment:
Yes, a new (optional) field is proposed [further down](https://github.com/NREL/EnergyPlus/pull/5921/files#diff-9bc0a8d4cebbce9ddf4c43d508ca2b2cR310) for Lights, refrigerated case, and airflow window property.